src/txn/txn_rollback_to_stable.c

```diff
@@ -1360,10 +1360,10 @@ __rollback_to_stable_btree_walk(WT_SESSION_IMPL *session, wt_timestamp_t rollbac
               WT_READ_NO_EVICT | WT_READ_WONT_NEED | WT_READ_VISIBLE_ALL)) == 0 &&
       ref != NULL) {
         __rollback_progress_msg(session, rollback_timer, 0, 0, &rollback_msg_count, true);
-        if (F_ISSET(ref, WT_REF_FLAG_INTERNAL))
+        if (F_ISSET(ref, WT_REF_FLAG_INTERNAL)) {
```

Comment:
I had to update this @kommiharibabu. This code is not present in the 7.0 branch, as in we are not doing a `WT_WITH_PAGE_INDEX`. Is that the right thing to do here?

Reply:
Yes, it is correct.

Reply:
I can't find when this code disappeared. Any reason why it changed? And should it be changed?

Reply:
With the support of writing the fast truncate information to the disk, the handling of RTS with the fast truncate data has changed that function and it led to the removal of that code completely from versions > 6.0.

Reply:
Thank you!